Firestore/Example/Tests/Local/FSTLocalSerializerTests.mm

```diff
@@ -66,8 +68,8 @@ @interface FSTLocalSerializerTests : XCTestCase
 @implementation FSTLocalSerializerTests
 
 - (void)setUp {
-  DatabaseId database_id("p", "d");
-  self.remoteSerializer = [[FSTSerializerBeta alloc] initWithDatabaseID:database_id];
+  self->databaseId = DatabaseId("p", "d");
```

Comment:
You can just assign to `_databaseID` directly. As I often tell @jdimond: you should never have to write `self->`.

Reply:
done